css/app.css

```diff
@@ -1,3 +1,820 @@
+  color: #fff;
+}
+
+.main-article-latest {
+  display: flex;
+  margin-top: 0.2rem;
+  background-color: #fcece3;
+  justify-content: space-between;
+}
+
+.main-article-latest-details {
+  padding: 0.625rem 0.875rem 0;
+}
+
+.main-article-latest-subtitle {
+  font-size: 0.8125rem;
+  color: #f42;
+}
+
+.main-article-latest-title {
+  font-size: 0.8125rem;
+  margin-top: 0.3125rem;
+  color: #f42;
+}
+
+.main-article-subarticles {
+  list-style-type: square;
+  padding-left: 1.25rem;
+}
+
+.main-article-subarticles li {
+  color: #f42;
+  padding: 0.375rem 00.375rem 0.875rem;
+  border-bottom: 1px rgba(170, 170, 170, 0.5) solid;
+}
+
+/* latest deals */
+
+.latest-deals-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.latest-deals-grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 1.25rem;
+}
+
+.latest-deals-item-image {
+  position: relative;
+  overflow: hidden;
+}
+
+.latest-deals-discount-badge {
+  position: absolute;
+  right: -3.75rem;
+  top: -1.875rem;
+  background-color: hsl(48deg 100% 50%);
+  padding: 24% 15% 2px;
+  transform: rotate(45deg);
+  font-size: 0.75rem;
+  font-weight: 600;
+}
+
+.latest-deals-item-subtitle {
+  font-size: 1rem;
+  color: #555;
+}
+
+.latest-deals-old-price {
+  font-size: 0.8125rem;
+  color: #aaa;
+  margin-right: 0.25rem;
+  text-decoration: line-through;
+}
+
+/* footer */
+
+.footer-social {
+  background-color: hsl(0deg 0% 7%);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 10rem;
+  padding: 2.5rem 0 2.5rem;
+}
+
+.footer-social-icons,
+.footer-links {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer-copyright {
+  background-color: #000;
+  text-align: center;
+  padding: 3.5625rem 0;
+  color: #aaa;
+  font-size: 0.8125rem;
+}
+
+/* Pseudo classes for the linters errors */
+
+.nav-main-right li:nth-child(2) a {
+  color: #f42;
+}
+
+.showcase-item-social li:not(:first-child) {
+  padding: 0 0.9375rem;
+}
+
+.showcase-item:nth-child(2) {
+  background: rgb(0, 0, 0);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/showcase-images/showcase-small-1.jpg);
+  background-position: center center;
+  background-size: cover;
+}
+
+.showcase-item:nth-child(3) {
+  background: rgb(0, 0, 0);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/showcase-images/showcase-small-2.jpg);
+  background-position: center center;
+  background-size: cover;
+}
+
+.latest-news-article-title:hover {
+  color: #f42;
+  text-decoration: underline;
+  text-decoration-color: rgba(255, 67, 34, 0.5);
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-primary{
  color:#f42;
}
```
**Html file**

```
<div class="color-primary">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_